src/it/warResources/src/main/webResource-filtering/help-globalConfig.html

```diff
@@ -0,0 +1,18 @@
+<div>
+  <p>
+    This HTML fragment will be injected into the configuration screen
+    when the user clicks the 'help' icon. See global.jelly for how the
+    form decides which page to load.
+    You can have any HTML fragment here.
+  </p>
+  <p>
+    You can also filter using interpolation like: 
+    \${prop1\} ${prop1}
+    or 
+    \@prop2\@ @prop2@
+    and these will be replaced at build time
+    it is even possible to use system properties
+    \@java.class.version\@ ${java.class.version}
```

Comment:
possibly not the best system property to use...
`java.vendor.url` could be better and checking the replacement starts with `https://` 

but as this currently does not work there is not much point in changing...